pkg/machine/e2e/list_test.go

```diff
@@ -135,6 +135,15 @@ var _ = Describe("podman machine list", func() {
 		var listResponse []*machine.ListReporter
 		err = jsoniter.Unmarshal(listSession.Bytes(), &listResponse)
 		Expect(err).To(BeNil())
+
+		// table format includes the header
+		list3 := new(listMachine)
+		listSession3, err3 := mb.setCmd(list.withFormat("table {{.Name}}")).run()
```

Comment:
OK, I'd love to give this a full thorough review and understand `listMachine` and all that, but, sorry, I'm not. (Please don't take it personally). My first thought on reading this is, did you really mean `setCmd(list`, or should that be `list3`? And that's why I hate hate hate defining new variables for everything. Too easy to mix & match.

So, should that be `list3`?

Reply:
I see what you mean. I can use `list` instead of `list3`. I just saw `list2 := new(listMachine)` above and I thought why not.